bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1254,7 +1254,9 @@ private void checkStatus() {
                 additionalErrors = List.of(e.getMessage());
                 localHash = baseHash;
             }
-            PullRequestCheckIssueVisitor visitor = checkablePullRequest.createVisitor(checkablePullRequest.targetHash());
+
+            Hash jcheckConfHash = checkablePullRequest.targetHash();
+            PullRequestCheckIssueVisitor visitor = checkablePullRequest.createVisitor(jcheckConfHash);
```

Comment:
Any particular reason to spell out the types here?

Reply:
Nope, thanks for catching